Engine/Engine/Core/InputManager.h

```diff
@@ -1,14 +1,14 @@
-/*#ifndef INPUTMANAGER_H_
+#ifndef INPUTMANAGER_H_
 #define INPUTMANAGER_H_
 
 #include <GLFW/glfw3.h>
-#include "Timer_Sad.h"
+//#include "Timer_Sad.h"
 #include <map>
 #include <Vector>
```

Comment:
vector should be lowercase here. Doesn't compile on Linux otherwise

Reply:
Removed InputManager.h & cpp from the project.